recipes/sqt/meta.yaml

```diff
@@ -0,0 +1,31 @@
+about:
+  home: https://bitbucket.org/marcelm/sqt
+  license: MIT
+  summary: 'Command-line tools for the analysis of high-throughput sequencing data'
+package:
+  name: sqt
+  version: "0.6.0"
+source:
+  fn: sqt-0.6.0.tar.gz
+  url: https://pypi.python.org/packages/source/s/sqt/sqt-0.6.0.tar.gz
+  md5: b1b9f6a1aedb3c462d87dc8de284497f
+requirements:
+  run:
+    - python >=3.3
```

Comment:
Yepp, just forgot about it. I even [did it for whatshap](https://github.com/bioconda/bioconda-recipes/commit/bfb202a637c5a1354b2959ae250151ce03739e21) already.